pkg/asset/agent/manifests/infraenv.go

```diff
@@ -70,6 +73,10 @@ func (i *InfraEnv) Generate(dependencies asset.Parents) error {
 		if installConfig.Config.Proxy != nil {
 			infraEnv.Spec.Proxy = getProxy(installConfig)
 		}
+
+		if agentConfig.Config != nil {
+			infraEnv.Spec.AdditionalNTPSources = agentConfig.Config.AdditionalNTPSources
```

Comment:
Do you think that any additional validation (on the content of `AdditionalNTPSources`) may be required/useful?